libraries/helpers.rb

```diff
@@ -570,6 +570,19 @@ def is_compute_node_bootstrap_timeout_updated?(previous_config, config)
   evaluate_compute_bootstrap_timeout(previous_config) != evaluate_compute_bootstrap_timeout(config)
 end
 
+def evaluate_realmemory_to_ec2memory_ratio(config)
+  require 'json'
+  extra_chef_attrs = config.dig("DevSettings", "Cookbook", "ExtraChefAttributes")
+  JSON.parse(extra_chef_attrs).dig("cluster", "realmemory_to_ec2memory_ratio")
```

Comment:
I believe that is the config does not have DevSettings/Cookbook, JSON.parse(nil) will produce an error